cybershell/validation_framework.py

```
@@ -0,0 +1,973 @@
+        
+        return min(1.0, max(0.0, base_confidence))
+    
+    def _generate_proof(self,
+                       vulnerability_type: str,
+                       evidence: List[ValidationEvidence],
+                       exploitation_result: Dict) -> Dict:
+        """Generate proof of exploitation"""
+        
+        proof = {
+            'vulnerability_type': vulnerability_type,
+            'timestamp': datetime.now().isoformat(),
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)